frontend/src/components/ImageViewerContainer/ImageViewerContainer.js

```diff
@@ -137,6 +146,21 @@ class ImageViewerContainer extends React.Component {
     }
   }
 
```

Comment:
Für diese beiden Methoden würde ich noch mehr Kommentare einfügen, die klar machen, welcher Teil genau dafür verantwortlich ist, dass das Bild die gleiche Grösse behält beim Verändern der Fenstergrösse. Wäre das zum Beispiel das Aufrufen von this.state.renderer.resetCamera() und this.vtkContext.renderWindow.render() in handleResize()?

Reply:
Für die beiden Funktionen sowie für weitere Teile im Component habe ich nun noch genauere Kommentare hinzugefügt.